### PR TITLE
"Super Mario" theme tweaks, more translation proofreading

### PR DESCRIPTION
**Pull request by jpmac26**
I now have a much bigger appreciation for GIMP ;)

### CODE REVIEW

File: Editor/languages/editor_en.ts [hunk 1]
```diff
@@ -6507,7 +6507,9 @@ The special state is defined by the active configuration pack.</translation>
         <source>Show a blank screen instead of loading screen.
 In the Legacy Engine level loading screen, it would otherwise 
 show the current playable character(s) and number of lives.</source>
-        <translation type="unfinished"></translation>
+        <translation>Show a blank screen instead of the standard loading screen.
+This will prevent the Legacy Engine from displaying the player&apos;s
```
Comment (Wohlstand) [inline Editor/languages/editor_en.ts]: This is not for legacy engine, this feature introduced in SMBX-38A. the description does describes what it's the pre-level intro scene with showing the legacy engine as example.

Comment (jpmac26) [inline Editor/languages/editor_en.ts]: Understood, I'll fix the translation in a moment. Thanks for the clarification!

**Comment by jpmac26** [comment]
Ok, the translation should be correct now.

**Comment by Wohlstand** [comment]
Also, the tip for a future:
Every time you are replacing source strings, please, replace them via Atom or Visual Studio Code, etc. with find&replace string in multiple files (language files, project files, etc.). Because every time you changing source string you are breaking other translations are using source string as key. That is required for the editor. For case of PGE Engine, it uses the TR-ID thing and you are able to edit English texts as you want without breaking translations (because the key is a special technical string which is same, and source string (the special comment) can be changed without breaking keys are referring translations)

**Pull request merged**